src/server.ts

```diff
@@ -1,31 +1,24 @@
 require('custom-env').env()
 import express from "express";
 import bodyParser from "body-parser";
-import EventRoute from "./routes/event-route";
+import routes from "./routes";
+import "reflect-metadata";
 
 const app = express();
+const cors = require('cors');
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-// added by me for testing
 app.get("/", (request, response) => {
   response.json({ message: "Server starts successfully!" });
 });
 
 // routes
-app.use("/api/v1/event", EventRoute);
+app.use("/api/v1", routes);
```

Comment:
A big plus on this as well. Centralized routes would make any future addition easier